### ISSUE TITLE
Take Photo View Controller has unreadable navigation title and button

### ISSUE THREAD

**Issue by dancarls0n** [onfido/onfido-ios-sdk#104]
### What was the expected behaviour?

Users can view back and title after choosing 'Passport' in Select a Document Screen'

### What happened instead?

Users see only a white bar, the back button is clickable, but not visible. The title is there, but has the same colour as the title bar

### Version info:

- Cocoapods (if applicable): Onfido (11.1.1)
- Onfido SDK (Debug/Release): Comined now? Running in debug mode, also happens when compiled for production
- iOS:  12.2
- Xcode: 10.2.1
- Device/Simulator: iPhone 8 (12.2)
- Device/Simulator language: English

### Integration configuration:
```

 let appearance = Appearance(primaryColor: Constants.Colors.Purple, primaryTitleColor: UIColor.white, primaryBackgroundPressedColor: .lightGray)
        let config = try! OnfidoConfig.builder()
            .withToken(Config.sharedInstance().getOnFidoAPIKey())
            .withApplicantId(applicant)
            .withDocumentStep()
            .withFaceStep(ofVariant: .photo)
            .withAppearance(appearance)
            .build()
        let onfidoFlow = OnfidoFlow(withConfiguration: config)
            .with(responseHandler: self.handleOnFidoResult)
        
        do {
            let onfidoRun = try onfidoFlow.run()
            presentingVC?.present(onfidoRun, animated: true, completion: nil)
        }
        catch let error { }
```

### Steps to reproduce:

<img width="473" alt="Screen Shot 2019-07-22 at 9 55 43 AM" src="https://user-images.githubusercontent.com/1753568/61638416-5b025800-ac67-11e9-93c8-b342fa420caa.png">

<img width="469" alt="Screen Shot 2019-07-22 at 9 50 24 AM" src="https://user-images.githubusercontent.com/1753568/61637950-715be400-ac66-11e9-925f-90c581ecfac8.png">

![Screen Shot 2019-07-22 at 9 52 02 AM](https://user-images.githubusercontent.com/1753568/61637948-715be400-ac66-11e9-9624-4bd27a324665.png)

**Comment by keremgund** [comment]
Hey @dancarls0n , do you make any global customisation on UIAppearance or UINavigationBar ? (i.e `UINavigationBar.appearance().tintColor = myColor`)
Also do you have custom navigationbar configuration for presentingVC ? I tested 11.1.1 version with 12.2 device but couldn't find any issue. Can you also try to run our sample app (which inside this repo) on the same device you found an issue and share results with us ?

Thanks,
Kerem

**Comment by dancarls0n** [comment]
It was these settings that caused the issue. 

```
UINavigationBar.appearance().backgroundColor = .white
UINavigationBar.appearance().tintColor = .purple
UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor:.black]
```

Removing these calls removed the white band. It seems like you override the tint color and titleTextAttributes. Perhaps you should set the backgroundColor explicitly when presenting the OnfidoNavigationController during photo taking?

**Comment by keremgund** [comment]
Hi @dancarls0n, yes seems like we can make improvements on our side to ensure that we show proper UI even integrator uses global appearance customisation. 
I created internal ticket to resolve this problem and will let you know once we release it.

**Comment by keremgund** [comment]
Hi @dancarls0n , just wanted to give you an update. At the moment we're working on the fix and we aim to release it soon.

Thank you, 
Kerem

**Comment by keremgund** [comment]
Hi @dancarls0n , we just released new version [14.0.0] which includes fix for this issue. 
Please upgrade to 14.0.0 version to get the fix and let us know if you have any question.

Thanks,
Kerem

**Comment by keremgund** [comment]
Closing the ticket due to lack of response. Please open new one if you think the problem still exists.

Thanks,
Kerem
